### ISSUE TITLE
STViT-Small training does not reach reported accuracy

### ISSUE THREAD

**Issue by chamely** [hhb072/STViT#11]
Hi, thank you for releasing the code and model for STViT.

I’ve been trying to train `stvit_small` with `224×224` input size by following the command provided in the README.

I'm using a single GPU (RTX 4090) and ran the following command:

```

python main.py --model stvit_small --data-path /path/to/imagenet --batch-size 256 --drop-path 0.1 --epoch 300 --dist-eval \
--output_dir ckpt --input-size 224 2>&1 | tee -a log.txt
```

After around 150 epochs, the top-1 accuracy is still around 73%, and it seems unlikely to reach the 83.6% reported in the paper even after 300 epochs.

Is there any important training setting I may have missed?

Could you release the training logs of STViT small?

I trying lr 0.001 too.

Thanks in advance for your help!

![Image](https://github.com/user-attachments/assets/c23f626c-6ab1-4368-982c-4d2673d558bd)
